### DISCUSSION TITLE
Open PRs

### DISCUSSION THREAD

**Discussion by ShaMan123** [fabricjs/fabric.js#7239] [General]
@asturur there are open PRs I submitted that are going stale.
Could we make an effort to finalize and merge them?
#7100 
#7141 
#7178 
#7034 

Thanks!

**Comment by asturur** [comment]
Yes i have been out two weeks for family leave didn't do much. we need to tackle some of those PRs.
#7178 i would like to understand how we can't get away with one repo, and why create 2/3 with 2/3 publishing needs.
I m not an expert of NPX but you didn't help me understanding either.
#7034 we are going to do it, but i m not sure that is how it should work. i assigned myself and try to handle it tomorrow morning.
#7141 i need to pull out an example of something i wanted to show for setting the position

**Reply by ShaMan123** [reply]
🤟

#7178 :
The usage with `create-react-app` is as follows:
```bash
npx create-react-app my-app --template [template-name]
```
In order to support both js/ts templates it is necessary to publish them separately AFAIK.
If you don't care about `npx create-react-app` and want it available only as a fabric script it is not necessary to publish the templates. This option is ready as well.
The main repo holds both templates and is in charge of building them for ease of maintenance and development. Developing the templates is done by working on the `common` folder and running `npm run build` to generate both templates.
This is explained in the repo's README.

Hope this is clear enough.
